ood-portal-generator/lib/ood_portal_generator/dex.rb

```diff
@@ -226,7 +240,7 @@ def frontend
     end
 
     def copy_ssl_certs
-      return if !ssl? || @view.ssl.nil? || ! tls_cert.nil? || ! tls_key.nil?
+      return if !enabled? || !ssl? || @view.ssl.nil? || ! tls_cert.nil? || ! tls_key.nil?
```

Comment:
I believe this is the only side affect after initialization. I've added a test case for the same - that is _doesn't_ copy ssl when it's not enabled.